CMakeLists.txt

```diff
@@ -331,6 +331,7 @@ endif (WIN32)
 if (CLR_CMAKE_PLATFORM_UNIX)
   add_definitions(-DPLATFORM_UNIX=1)
   add_definitions(-DFEATURE_PAL_SXS)
+  add_definitions(-DFEATURE_COREFX_GLOBALIZATION)
```

Comment:
If there's a better place to define this, let me know. I see it defined in another makefile, but that doesn't appear to be affecting all relevant areas.